images_test.go

```diff
@@ -117,6 +89,8 @@ func TestImagesListOk(t *testing.T) {
 	}
 }
 
+// Special tests for the IMAGES command.
+
```

Comment:
These comments are there to group a set of test functions, or to divide a test file between different (but related) commands. In the case you pointed out, it serves for grouping two test functions.